thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/client/ThirdEyeCacheRegistry.java

```diff
@@ -186,15 +186,16 @@ private static void initPeriodicCacheRefresh() {
     // as weeklyService starts before hourlyService finishes,
     // causing NPE in reading collectionsCache
 
-    cacheResource.refreshCollections();
-
-    cacheResource.refreshDatasetConfigCache();
-    cacheResource.refreshMetricConfigCache();
-    cacheResource.refreshDashoardConfigsCache();
-
-    cacheResource.refreshMaxDataTimeCache();
-    cacheResource.refreshDimensionFiltersCache();
-    cacheResource.refreshDashboardsCache();
+    // Start initial cache loading asynchronously to reduce application start time
+    Executors.newSingleThreadExecutor().submit(() -> {
+      cacheResource.refreshCollections();
+      cacheResource.refreshDatasetConfigCache();
+      cacheResource.refreshDashoardConfigsCache();
+      cacheResource.refreshDashboardsCache();
+      cacheResource.refreshDimensionFiltersCache();
+      cacheResource.refreshMetricConfigCache();
+      cacheResource.refreshMaxDataTimeCache();
```

Comment:
moving maxDataTimeCache to the end will result in the dashboard to load very slowly in the beginning. Any strong reason to shift it down?